### PR TITLE
Add guest house room booking component for home screen

### PR DESCRIPTION
**Pull request by adityachopra29**

### CODE REVIEW

**Comment by adityachopra29** [comment]
Screenshot 2024-03-20 at 11 13 49 PM

**Comment by adityachopra29** [comment]
@beingnoble03

File: src/components/room-new/index.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+            Book here
+            </span>
+            <button className={tailwindWrapper('bg-[#133BC5] self-start rounded-md text-white px-28 py-3')} onClick={() => {
+              window.alert("book button clicked")
+            }}>Book here</button>
+        </div>
+
+    </>) 
+    :
+      (<>
```
Comment (beingnoble03) [inline src/components/room-new/index.js:86]: remove `<>`, not req.

File: src/components/room-new/index.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        <div className={tailwindWrapper('')}>
```
Comment (beingnoble03) [inline src/components/room-new/index.js:87]:
```suggestion
        <div>
```

File: src/components/room-new/index.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    
+    return(
+      <>
+      {bookingRequests.results && bookingRequests.results.length == 0
+    ?(<>
+        <div className={tailwindWrapper('flex')}>
+          <div className={tailwindWrapper('text-[#133BC5] mt-20 text-2xl font-semibold')}>Guest House</div>
+        </div>
+        <div className={tailwindWrapper(' my-8')}>Room Bookings</div>
+        <div className={tailwindWrapper('flex  flex-col gap-5')}>
```
Comment (beingnoble03) [inline src/components/room-new/index.js:55]:
```suggestion
        <div className={tailwindWrapper('flex flex-col gap-5')}>
```

File: src/components/room-new/index.js [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    const { activePage, loading } = this.state;
+    
+    return(
+      <>
+      {bookingRequests.results && bookingRequests.results.length == 0
+    ?(<>
+        <div className={tailwindWrapper('flex')}>
+          <div className={tailwindWrapper('text-[#133BC5] mt-20 text-2xl font-semibold')}>Guest House</div>
+        </div>
+        <div className={tailwindWrapper(' my-8')}>Room Bookings</div>
```
Comment (beingnoble03) [inline src/components/room-new/index.js:54]:
```suggestion
        <div className={tailwindWrapper('my-8')}>Room Bookings</div>
```

**Review by beingnoble03 — CHANGES_REQUESTED** [review]
also, add padding-bottom in the main component. there's no padding between the 2 components, as visible in the ss.

**Pull request closed without merging**